plugins/source/gcp/resources/services/resourcemanager/tag_keys.go

```diff
@@ -0,0 +1,59 @@
+package resourcemanager
+
+import (
+	resourcemanager "cloud.google.com/go/resourcemanager/apiv3"
+	pb "cloud.google.com/go/resourcemanager/apiv3/resourcemanagerpb"
+	"context"
+	"github.com/apache/arrow/go/v14/arrow"
+	"github.com/cloudquery/cloudquery/plugins/source/gcp/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+	"google.golang.org/api/iterator"
+)
+
+func TagKeys() *schema.Table {
+	return &schema.Table{
+		Name:        "gcp_resourcemanager_tag_keys",
```

Comment:
Should we call this table `gcp_resourcemanager_project_tag_keys`? That would make it consistent with `gcp_resourcemanager_organization_tag_values`. If so we should rename `TagKeys` to `ProjectTagKeys`

Reply:
Updated to use `organization` and `project` prefixes for consistency.